logs/querysample/querysample.go

```diff
@@ -51,38 +51,55 @@ func transformExplainJSONToQuerySample(logLine state.LogLine, explainText string
 		ExplainSource:     pganalyze_collector.QuerySample_AUTO_EXPLAIN_EXPLAIN_SOURCE,
 		ExplainFormat:     pganalyze_collector.QuerySample_JSON_EXPLAIN_FORMAT,
 		ExplainOutputJSON: &explainJSONOutput,
+		Parameters:        findQueryParameters(explainJSONOutput.QueryParameters),
 	}, nil
 }
 
 var autoExplainTextPlanDetailsRegexp = regexp.MustCompile(`^Query Text: (.+)\s+([\s\S]+)`)
 var herokuAutoExplainWithTabRegexp = regexp.MustCompile(`^Query Text: ([^\t]+)\t([\s\S]+)`)
+var autoExplainTextWithQueryParametersRegexp = regexp.MustCompile(`^Query Text: ([\s\S]+)\r?\n\s*Query Parameters: (.+)\r?\n\s*([\s\S]+)`)
+var autoExplainTextWithCostsRegexp = regexp.MustCompile(`^Query Text: ([\s\S]+?)\r?\n\s*([\S ]+  \(cost=\d+\.\d{2}\.\.\d+\.\d{2} rows=\d+ width=\d+\)[\s\S]+)`)
```

Comment:
Agree that it's easier to follow, I'll update 👍 Just a background, I did `{2}` since Postgres is using  `(cost=%.2f..%.2f rows=%.0f width=%d)`.

https://github.com/postgres/postgres/blob/4d93bbd4e0d414a33521f62d6249ac88486c866f/src/backend/commands/explain.c#L1815